ci/travis/37-latest-conda-forge.yaml

```diff
@@ -22,3 +22,5 @@ dependencies:
   - SQLalchemy
   - psycopg2
   - libspatialite
+  - pip:
+    - mapclassify
```

Comment:
For the conda-forge based environments, `mapclassify` should be available to install with conda, so you don't need to put it in the pip section

Reply:
And you can also remove `pysal` from all the environments as well, I think

Reply:
pysal is used in tests to get example Shapefile for classification. I'll update those tests as well to suite new PySAL scheme.